back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -31,9 +31,11 @@
       parameter :depth, 'Filter by depth (AND)', required: false
       parameter :topics, 'Filter by topics (AND)', required: false
       parameter :areas, 'Filter by areas (AND)', required: false
-      parameter :folder, "Filter by folder (project folder id)", required: false
       parameter :publication_statuses, "Return only publications with the specified publication statuses (i.e. given an array of publication statuses); always includes folders; returns all publications by default", required: false
-      parameter :remove_childless_parents, 'Use the visibility rules of children on the parent and remove the empty ones', required: false
```

Comment:
I'm confused why this parameter was already present before this change. To me the diff in `back/app/services/admin_publications_filtering_service.rb` looks like we just introduced it in this PR? :thinking:

Reply:
Yes. I believe it had existed at some time in the past, then was removed but the parameter declaration in the test was not (easy to miss as doesn't cause any issues). I then recreated a filter using the same name, and therefore left the parameter declaration in the test.